### PR TITLE
Add simple Rust TCP client example

### PR DESCRIPTION
**Pull request by muhammad-owais-javed**
This is an example of Rust TCP client for the "HTTP client example #48 "

Signed-off-by: Owais <muhammad.owais.javed.1417@gmail.com>

### CODE REVIEW

File: Rust/simple-tcp-client/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[package]
+name = "tcpclient"
+version = "0.0.1"
```
Comment (rvolosatovs) [inline Rust/simple-tcp-client/Cargo.toml:3]: Please use `0.1.0` for consistency with other examples

File: Rust/simple-tcp-client/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+[package]
+name = "tcpclient"
```
Comment (rvolosatovs) [inline Rust/simple-tcp-client/Cargo.toml:2]:
```suggestion
name = "std-tcp-client"
```

**Review by rvolosatovs — APPROVED** [review]
Please move the example to `std-tcp-client` and adjust crate name and version. Apart from that, fantastic job, thank you for contribution!

File: Rust/simple-tcp-client/Cargo.toml (hunk 1 above)
Comment (muhammad-owais-javed) [reply Rust/simple-tcp-client/Cargo.toml]: Crate name and version has been changed now.

**Review by rvolosatovs — APPROVED** [review]
Stellar work, thank you!

**Pull request merged**